DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -378,7 +378,13 @@ public IEnumerable<DelegateCourseInfo> GetDelegateCourseInfosForCourse(int custo
         {
             return connection.Query<DelegateCourseInfo>(
                 $@"{selectDelegateCourseInfoQuery}
-                    WHERE cu.CentreID = @centreId
+                    WHERE (cu.CentreID = @centreId OR
+                            (cu.AllCentres = 1 AND
+                                EXISTS (SELECT CentreApplicationID
+                                        FROM CentreApplications
+                                        WHERE ApplicationID = cu.ApplicationID AND

```

Comment:
Is there any possibility of this being confused with `ApplicationID` from `Applications` in the `selectDelegateCourseInfoQuery`? Probably not, since it presumably works, but I think it's worth explicitly disambiguating on general principle.